### ISSUE TITLE
Gitter window restore down not working properly in windows 10 vian desktop application

### ISSUE THREAD

**Issue by apalm1341** [gitterHQ/gitter#1543]
I have gitter desktop app on windows 10 computer. I clicked it twice from desktop. Then, it opened up afterwards full screen size. Then, when I click on restore down button on gitter desktop app window, I get a weird glitch with gitter where it doesn't resize window properly like it normally has done for me in the past. It shows up really small, and I have to manually readjust the size myself. Is there a way I can fix this so that the default should be showing up instead of a really small window?

**Comment by InternetUnexplorer** [comment]
Just to clarify:
1. You open the desktop app
2. It appears maximized
3. You minimize it
4. When you restore the window by clicking on it in the task bar...
5. It is no longer maximized

Is that correct? If so, I tried reproducing the issue on my computer (running Windows 10 with the desktop app) and did not see the behaviour you have described, it worked exactly as expected, and the window appeared maximized when restored. Trying this on another computer (also running Windows 10 with a fresh installation of the desktop app) yielded the same results.

It seems to be a problem only on your end, and not a problem with the desktop app. If the problem continues to be a frustration for you I would recommend using Gitter in your browser for the time being.

**Comment by MadLittleMods** [comment]
Sounds related to https://github.com/gitterHQ/desktop/issues/84 which I was not able to reproduce on Windows 10.

**Comment by apalm1341** [comment]
Ya it occurs right after I double click the gitter desktop app on my computer. Right after this, the program runs and full screen size gitter window appears. Then, when I force down it, not minimize, it doesn't resize properly sometimes and shows up too small and then I have to manually readjust the size of the window myself.

**Comment by apalm1341** [comment]
I think sometimes I can force down it to normal size myself eventually if the program is still on the taskbar.

**Comment by apalm1341** [comment]
I wish I could upload some images to show u guys, but had trouble couple days ago when trying to figure that out.

**Comment by apalm1341** [comment]
Dragging and dropping files using sharex program and/or snippet tool does not work for me. Copying and pasting doesn't work either.

**Comment by MadLittleMods** [comment]
@apalm1341 Please condense your replies down into a single reply.

When you say "restore down button" or "force down it, not minimize", can you explain what this is? I am not famiilar with what you could be describing. 

You can upload your images https://imgur.com/ if you can' figure it out on GitHub issues. A gif showing off a reproduction would be great.

**Comment by apalm1341** [comment]
I apologize. I meant to say restore down previously and not force down. I am talking about the button that has two rectangles side by side on a typical window. I'll try and see if I can figure out how to use imgur also to post my images.

**Comment by MadLittleMods** [comment]
For reference,

![](http://i.imgur.com/DH8DJHe.png)

**Comment by apalm1341** [comment]
Yes, that restore button is what I have been talking about. Thanks for posting that pic.

**Comment by apalm1341** [comment]
I think sometimes they call it restore down also. Maybe I am wrong though.

**Comment by C-F-4** [comment]
Restore is a button used to restore the size of a window when you are in Full-Screen mode. Try Restore Button when you're in Full-Screen Mode. It should work fine. It won't, however, work if you're not in a Full-Screen Mode. That's why it's disabled. @apalm1341

**Comment by apalm1341** [comment]
![restore down issue pic](https://cloud.githubusercontent.com/assets/20910742/20779959/fdd7db8c-b745-11e6-9f7c-9daab0c10416.GIF)
